src/battle/190B20.c

```diff
@@ -643,20 +643,20 @@ void func_80263268(void) {
     }
 }
 
-void func_80263300(void) {
+void btl_init_menu_items(void) {
```

Comment:
Assuming all this stuff will get backported... can we make an issue or something to track these

Reply:
Gonna diff enums, common_structs, and symbol_addrs.